plugin-server/src/worker/ingestion/properties-updater.ts

```diff
@@ -98,7 +88,7 @@ export async function upsertGroup(
             const group: Group | undefined = await db.fetchGroup(teamId, groupTypeIndex, groupKey, client, {
                 forUpdate: true,
             })
-            const createdAt = group?.created_at || timestamp
+            const createdAt = DateTime.min(group?.created_at || DateTime.now(), timestamp)
```

Comment:
Is `group?.created_at` of type `DateTime`?

Reply:
yes based on types here: https://github.com/PostHog/posthog/blob/master/plugin-server/src/types.ts#L533